crates/rpc/rpc/src/layers/jwt_secret.rs

```diff
@@ -79,26 +76,24 @@ impl JwtSecret {
     /// Tries to load a [`JwtSecret`] from the specified file path.
     /// I/O or secret validation errors might occur during read operations in the form of
     /// a [`JwtError`].
-    pub fn from_file(fpath: &Path) -> Result<Self, JwtError> {
-        let hex = std::fs::read_to_string(fpath)
-            .map_err(|err| JwtError::IORead { err, path: fpath.to_path_buf() })?;
+    pub fn from_file(fpath: &Path) -> eyre::Result<Self> {
```

Comment:
I want to keep JwtError and not use eyre, see comment above

Reply:
Sure, this is done.